web/static/css/_comments.scss

```diff
@@ -10,7 +10,7 @@
 }
 
 .comment-loading {
-  @include animation(loading-spinner 0.6s infinite linear);
+  animation: 0.6s linear infinite loading-spinner;
```

Comment:
Since 0.6s is used twice we should define it as a variable. How about something like `$long-duration: 0.6s` in `_variables.scss`?

Reply:
then `animation: $long-duration linear infinite loading-spinner;`

Reply:
Good idea. Extracted in 75bef98668c7d6429e4e593883c6640672913703